src/components/ItemList.svelte

```diff
@@ -111,6 +112,10 @@
         <input type="checkbox" bind:checked={showExpired} />
         Show expired metrics
       </label>
+      <label>
+        <input type="checkbox" bind:checked={paginated} />
+        Paginate
+      </label>
```

Comment:
- I opted to put this checkbox on top instead of below the Pagination bar because I thought that if the user unclicked this box but changed their mind, they would not have to scroll past hundreds of items to get to the bottom to find it. 

- "Paginate" instead of "Show All" because next to the "Show expired metrics" box, "Show All" might not be as clear a name as "Paginate"? 

What do you think @wlach?